adops/gam-creative-banner-sbs.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: page_v2
+title: GAM Step by Step - Banner/Outstream/AMP Creatives
+head_title: GAM Step by Step - Banner/Outstream/AMP Creatives
+description: Set up banner and outstream creatives for Prebid in Google Ad Manager.
+pid: 2
+sidebarType: 3
+---
+
+
+# GAM Step by Step - Banner/Outstream/AMP Creatives
+
+This page walks you through the steps required to create banner and outstream creatives to attach to your Prebid line items in Google Ad Manager (GAM).
+
+{: .alert.alert-success :}
+For complete instructions on setting up Prebid line items in Google Ad Manager, see [Google Ad Manager with Prebid Step by Step](/adops/step-by-step.html).
+
+
+1. In GAM, select **Delivery** > **Creatives**.
+2. Under the **Display creatives** tab, click **New Creative**.
+3. Select your advertiser, then click **Third party**.
+4. Enter a **Name** for your creative. For example, `Prebid – banner – 1x1`.
```

Comment:
Would suggest adding the `-1` to the example creative name since we suggest making multiple copies.